tests/cupy_tests/linalg_tests/test_decomposition.py

```diff
@@ -70,8 +73,15 @@ def test_mode(self):
 @testing.gpu
 class TestSVD(unittest.TestCase):
 
+    def setUp(self):
+        global seed_testsvd
+        self.seed = seed_testsvd
+        seed_testsvd += 1
+
     @testing.for_float_dtypes(no_float16=True)
-    def check_usv(self, array, dtype):
+    def check_usv(self, shape, dtype):
```

Comment:
Can you test with complex dtypes?

Reply:
`for_float_dtypes` tests complex dtypes by default.

Reply:
wow, thank you